src/package/nuspec/TestPlatform.ObjectModel.nuspec

```diff
@@ -14,18 +14,13 @@
     <projectUrl>https://github.com/microsoft/vstest/</projectUrl>
     <copyright>© Microsoft Corporation. All rights reserved.</copyright>
     <tags>vstest visual-studio unittest testplatform mstest microsoft test testing</tags>
-    <repository type="git" 
-                url="https://github.com/microsoft/vstest" 
-                branch="$BranchName$" 
+    <repository type="git"
+                url="https://github.com/microsoft/vstest"
+                branch="$BranchName$"
                 commit="$CommitId$" />
-                
+
     <dependencies>
-      <group targetFramework="net45">
```

Comment:
I think we agreed that ObjectModel and the two depending libraries should stay supporting net462 so we don't immediately break nunit?

Reply:
Maybe make a new branch from this commit so we can go back to this later and not lose those changes??